diagnostic.md

```diff
@@ -48,11 +49,11 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    params.movie_id
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+    {{outlet}}
```

Comment:
The outlet is where Ember will render the template that is specified.  How we reference that data is through the Model.